src/app/globals.css

```diff
@@ -52,6 +52,13 @@
   font-weight: bold;
 }
 
+@font-face {
```

Comment:
Please remove this declaration as it's already defined using a woff font on line 34

Reply:
removed